src/Nethermind/Nethermind.Optimism.Test/Create2DeployerContractRewriterTests.cs

```diff
@@ -40,7 +36,7 @@ public void Test_Create2DeployerContractRewriter_WorksForNonExistentAccount()
 
         rewriter.RewriteContract(blockTree.FindHeader(1, BlockTreeLookupOptions.None)!, ws);
 
-        byte[] setCode = ws.GetCode(specHelper.Create2DeployerAddress!);
-        Assert.That(anyCode, Is.EquivalentTo(setCode));
+        byte[] setCode = ws.GetCode(PreInstalls.Create2Deployer);
+        Assert.That(setCode, Is.Not.Empty);
```

Comment:
Why did we change this test?

Reply:
Because we use the constant of the `PreInstalls.Create2Deployer` now that is defined in the spec and not the arbitrary address. The test ensure that the specific address defined as a const is not empty.